### PR TITLE
fix: Show expiration date on rentals and orders

### LINKED ISSUE #1141: Expiration is missing on the items details on parcels and estates
## 🎉 Description

We're missing the order and rental expiration date on the detail box of the `ParcelDetail` and `EstateDetail` components.

* [x] 🐛 This is a bug report.
* [ ] 📈 This is a feature request.

### PR DESCRIPTION
**Pull request by LautaroPetaccio**
This PR adds the missing order and rental expiration dates in the details.
Closes #1141

### CODE REVIEW

File: webapp/src/components/ManageAssetPage/Details/Details.tsx [hunk 1]
```diff
@@ -9,6 +9,7 @@ import { Box } from '../../AssetBrowse/Box'
 import { LinkedProfile } from '../../LinkedProfile'
 import { Props } from './Details.types'
 import styles from './Details.module.css'
+import formatDistanceToNow from 'date-fns/formatDistanceToNow'
```
Comment (juanmahidalgo) [inline webapp/src/components/ManageAssetPage/Details/Details.tsx:12]: can we move this import a few lines above? 🙏

File: webapp/src/modules/features/selectors.ts [hunk 2]
```diff
@@ -26,10 +26,12 @@ export const getIsMaintenanceEnabled = (state: RootState) => {
...
-    return getIsFeatureEnabled(
-      state,
-      ApplicationName.BUILDER,
-      FeatureName.RENTALS
+    return (
+      getIsFeatureEnabled(
+        state,
+        ApplicationName.BUILDER,
+        FeatureName.RENTALS
+      ) || true
```
Comment (juanmahidalgo) [inline webapp/src/modules/features/selectors.ts:34]: Is this intentional? I might be missing some context

**Review by juanmahidalgo — APPROVED** [review]
2 small comments/questions but approving anyways, looks great 👍

File: webapp/src/modules/features/selectors.ts (hunk 2 above)
Comment (LautaroPetaccio) [reply webapp/src/modules/features/selectors.ts]: Ops, no, it was a mistake, removed it!

File: webapp/src/components/ManageAssetPage/Details/Details.tsx (hunk 1 above)
Comment (LautaroPetaccio) [reply webapp/src/components/ManageAssetPage/Details/Details.tsx]: Moved it up!

**Pull request merged**